packages/shorebird_cli/lib/src/commands/build_command.dart

```diff
@@ -1,7 +1,9 @@
 import 'dart:io';
 
 import 'package:mason_logger/mason_logger.dart';
+import 'package:meta/meta.dart';
 import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/doctor/validators/shorebird_flutter_validator.dart';
```

Comment:
minor but should we move these validators out of the `doctor` directory since they're not tied to a specific command?

Reply:
Good call, done.